lib/gc/Dialect/OneDNNGraph/OneDNNGraphOps.cpp

```diff
@@ -65,11 +27,13 @@ LogicalResult onednn_graph::AddOp::inferReturnTypeComponents(
   auto getShapeIdx = [](ValueShapeRange operands, size_t i) {
     return operands.getShape(i);
   };
-  auto ret =
-      inferBroadcastShape<ValueShapeRange>(operands, outShape, getShapeIdx);
+  llvm::SmallVector<int64_t> input1, input2;
+  getShapeIdx(operands, 0).getDims(input1);
```

Comment:
Can we use something like `getShape` to get the ArrayRef of shape for performance?

Reply:
Thanks~ Optimized the code~

Reply:
Also, maybe `dyn_cast<ShapedType>()` should be better, in case we need to support other shaped types in the future.